CoronaCare/src/screens/AfterLogin.js

```diff
@@ -1,16 +1,31 @@
 import React, {Component} from 'react';
 import {View, TextInput, Text, Button, StyleSheet} from 'react-native';
 import Colors from 'react-native/Libraries/NewAppScreen';
-
+import { Navigation } from 'react-native-navigation';
 
 export default class Home extends Component {
+    constructor(props) {
+        super(props);
+    }
+    
+    changeToCameraScreen = () => {
+        Navigation.setRoot({
```

Comment:
this is wrong, should not reseting the root of the navigation stack. What you are trying to do here is a Navigation.push